File: example/php/generate.php
<?php

include('OOTFY.class.php');

/*** Configuration ***/
define('DOMAIN', ''); //Domain key
define('SIZE', 4); //Key size
define('PERIOD', 30); //Period
define('USER', ''); //User info
define('PIN', '0000'); //User PIN
/*** End ***/

$OOTFY = new OOTFY(DOMAIN, USER);
$OOTFY->set_keysize(SIZE);
$OOTFY->set_period(PERIOD);

echo 'Generated Token : '.$OOTFY->calcul_token(PIN);
echo "\n\n";

?>

File: example/python/generate.py
from OOTFY import OOTFY

if __name__ == "__main__":
    ### Configuration ####
    DOMAIN = 'example.domain.com' #Domain key
    SIZE   = 4        #Key size
    PERIOD = 30       #Period
    USER   = 'MyUser' #User info
    PIN    = '1234'   #User PIN
    
    OOTFY = OOTFY(DOMAIN, USER);
    OOTFY.set_keysize(SIZE);
    OOTFY.set_period(PERIOD);

    print('Generated Token : '+OOTFY.calcul_token(PIN)+'\n')
